contrib/navidrome.service

```diff
@@ -13,6 +12,7 @@ User=navidrome
 Group=navidrome
 Type=simple
 ExecStart=/usr/bin/navidrome
+StateDirectory=navidrome
```

Comment:
Should this be an absolute path?

Reply:
No, it's a relative path relative to `/var/lib` (see https://www.freedesktop.org/software/systemd/man/systemd.exec.html#RuntimeDirectory=)